numpy/__init__.pxd

```diff
@@ -126,6 +126,7 @@ cdef extern from "numpy/arrayobject.h":
         NPY_OBJECT
         NPY_STRING
         NPY_UNICODE
+        NPY_VSTRING
```

Comment:
Why not add the npystring API to this header too? Are they not being kept in sync?

Reply:
Added below - I was under the impression Cython<3 support would end soon, but there's no reason not to add them here.